fastcrypto/src/traits.rs

```diff
@@ -188,13 +180,20 @@ pub trait SigningKey: ToFromBytes + Serialize + DeserializeOwned + Send + Sync +
 /// to the ones on its associated types for private key and public key material.
 ///
 pub trait Authenticator:
-    signature::Signature + Display + Serialize + DeserializeOwned + Send + Sync + 'static + Clone
+    ToFromBytes + Display + Serialize + DeserializeOwned + Send + Sync + 'static + Clone
```

Comment:
do we really need this in the trait?

Reply:
Do you mean the `ToFromBytes` trait? The `as_bytes` function (which calls `as_ref`) is definitely being called a lot of places, so I think we need it until we introduce a cached object of similar. Or what do you suggest?

Reply:
let's leave it for another pr